apollo-ios-codegen/Package.swift

```diff
@@ -19,10 +19,10 @@ let package = Package(
       .upToNextMajor(from: "1.0.0")),
     .package(
       url: "https://github.com/apple/swift-collections",
-      .upToNextMajor(from: "1.0.6")),
+      .upToNextMajor(from: "1.1.0")),
```

Comment:
I'm on the fence about whether we should include these here or not? I could easily be persuaded to revert this change.